lib/ui/localization/export.dart

```diff
@@ -104,13 +106,15 @@ see your mnemonic will most surely end in them stealing your account and all
 your funds so proceed with caution.
 """;
   String get securityLoginBiometrics => """
-If you understand the risks of viewing your mnemonic, please tap below to
-start the authentication process.
+If you understand the risks of viewing your mnemonic, please check the below
```

Comment:
the wording is a little bit weird. Maybe `please select the checkbox below to start the authentication process`